buildpipeline/DotNet-CoreClr-Trusted-Linux.json

```diff
@@ -289,6 +291,7 @@
       "alwaysRun": true,
       "displayName": "Publish Artifact: BuildLogs",
       "timeoutInMinutes": 0,
+      "condition": "succeededOrFailed()",
```

Comment:
What's the difference between `succeededOrFailed()` and `always()` ? Just out of curiosity

Reply:
succeededorfailed doesn't include cancelled.  always does